### DISCUSSION TITLE
It can’t work on my MacBook Pro 2011

### DISCUSSION THREAD

**Discussion by Andylaw000** [chris1111/Wireless-USB-Big-Sur-Adapter#227] [Q&A]
I use OCL to install 12.6 on my unsupported MacBook Pro 2011.my slo status is unkept.I can’t disable it. I can install wireless driver ,plug  in Comfast cf811ac it can’t work , in the icon shows a Red Cross no usb device plug in. what can I do.thx

**Comment by chris1111** [comment]
Read the rule about [discussion ](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/discussions/33)
This is an issue not a discussion, this will be deleted
